src/client/event.rs

```diff
@@ -128,6 +128,7 @@ pub enum ClientEvent {
         /// Sequence number
         sequence: VarInt,
     },
+    ResourcePackStatus(ResourcePackC2s),
```

Comment:
We should reexport the ResourcePackC2s enum here because stuff in the `protocol` module is private/hidden.

```rust
pub use crate::protocol::packets::c2s::play::ResourcePackC2s as ResourcePackStatus;
```